docs/concepts/llm_basic_evaluation.md

```diff
@@ -262,6 +262,46 @@ qa = QuestionAnswerer(query=query).extract(retries=retries)
 
 ```
 
-We also put a hard stop after 3 attempts so we limit the number of attempts. If we’re still getting a `ValidationError` after multiple attempts, this indicates that we likely need to update our prompt rather than just retrying indefinitely. 
+We also put a hard stop after 3 attempts so we limit the number of attempts. If we’re still getting a `ValidationError` after multiple attempts, this indicates that we likely need to update our prompt rather than just retrying indefinitely.
 
-Although receiving a toxic answer from these models is unlikely due to their instruction fine-tuning, other issues such as hallucinations can occur more frequently.
+### Adding Multiple LLM calls to the validator
+
+To work with various LLM providers, you might consider using a generic prompt template. Mirascope offers a convenient classmethod `from_prompt` that takes all the class variables and fields from a prompt and creates a new model from a different provider. The example below demonstrates how you can assemble a jury of LLM providers to determine whether the given language is considered toxic:
+
+```python
+from mirascope.anthropic import AnthropicExtractor, AnthropicCallParams
+def validate_toxicity(generation: str) -> str:
+    """Check if the generated content language is toxic."""
+    jury: list[
+        tuple[
```

Comment:
Can we just type this as `tuple[type[BaseExtractor], BaseCallParams]`?

Reply:
oh yeah..i had that in my example but not here for some reason